packages/web/pages/assets/[denom].tsx

```diff
@@ -616,32 +594,9 @@ export const getStaticProps: GetStaticProps<AssetInfoPageProps> = async ({
    * Lookup for the current token
    */
   const token = currencies.find(
-    (currency) =>
-      currency.coinDenom.toUpperCase() === tokenDenom.toLocaleUpperCase()
+    (currency) => currency.coinDenom.toUpperCase() === tokenDenom.toUpperCase()
```

Comment:
Is it possible to use `getAsset` here? Instead of processing the chain list? We could make this change later when we get to migrating the asset detail page fully to tRPC but it's worth a mention and your thoughts.

Reply:
Done

Reply:
Nice!